libdevcore/Log.cpp

```diff
@@ -155,5 +155,7 @@ void dev::setThreadName(char const* _n)
 
 void dev::simpleDebugOut(std::string const& _s)
 {
-	std::cerr << _s << '\n';
+        string s = _s;
```

Comment:
If you need a copy, make it by passing the argument by value.

Reply:
Ok, but what's the difference? Still ends up with a copy.

Reply:
Wanted the copy code invoked in one place, not with every call.

Changed

cout << a << b;

to

cout << a + b;